lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -1669,6 +1669,32 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
       return b.create<arith::MulIOp>(loc, lhs, rhs);
     }
   }
+  if (auto gtTensor = dyn_cast<AtenGtTensorOp>(op)) {
+    AtenGtTensorOp::Adaptor adaptor(operands);
+    Type lhsDtype = payloadArgs[0].getType();
+    Type rhsDtype = payloadArgs[1].getType();
+
+    Type elementalType =
+        gtTensor.self().getType().cast<BaseTensorType>().getDtype();
```

Comment:
I guess you don't have to compute this `elementType`, you can use`lhsDtype` or `rhsDtype`.

Reply:
The problem here is signless integers; when you remain in torch dialect you have the knowledge of whether the integer is signed or not, but when you some to `linalg on tensors` world the integer becomes signless. This has to be looked into.